birdhouse/config/proxy/default.env

```diff
@@ -0,0 +1,39 @@
+# All env in this default.env can be overridden by env.local.
+
+# Folder inside "proxy" container to drop extra monitoring config
+export CANARIE_MONITORING_EXTRA_CONF_DIR="/conf.d"
+
+# Timeout for reading a response from the proxied server.
+# Any WPS processes taking longer than this should use async mode.
+export PROXY_READ_TIMEOUT_VALUE="240s"
+
+# Content of "location /" in file config/proxy/conf.d/all-services.include.template
+# Useful to have a custom homepage.
+export PROXY_ROOT_LOCATION="return 302 https://\$host/jupyter/hub/login;"
+
+export INCLUDE_FOR_PORT_80='$([ x"$ALLOW_UNSECURE_HTTP" = x"True" ] && echo "include /etc/nginx/conf.d/all-services.include;" || echo "include /etc/nginx/conf.d/redirect-to-https.include;")'
+
+export DELAYED_EVAL="
+  $DELAYED_EVAL
+  INCLUDE_FOR_PORT_80
+"
+
+# add any new variables not already in 'VARS' or 'OPTIONAL_VARS' that must be replaced in templates here
+export VARS="
+  $VARS
+  \$BIRDHOUSE_DEPLOY_COMPONENTS_JSON
+"
+
+export OPTIONAL_VARS="
+  $OPTIONAL_VARS
+  \$INCLUDE_FOR_PORT_80
+  \$PROXY_READ_TIMEOUT_VALUE
+  \$PROXY_ROOT_LOCATION
+"
+
+# add any component that this component requires to run
+COMPONENT_DEPENDENCIES="
+  $COMPONENT_DEPENDENCIES
+  ./config/canarie-api
+  ./config/twitcher
```

Comment:
Typo from previous edit?  I do not think proxy needs twitcher.  Both of them can function independently but they are just nicer together.  It's not the same level of coupling like between twitcher and magpie.

Reply:
BLOCKING